src/graphql.rs

```diff
@@ -593,22 +593,25 @@ impl AgentManager for MockAgentManager {
         &self,
         _networks: &HostNetworkGroup,
     ) -> Result<Vec<String>, anyhow::Error> {
-        Ok(vec!["hog@hostA".to_string()])
+        Ok(vec!["semi_supervised@hostA".to_string()])
```

Comment:
`semi-supervised` is a definitely better option.

Reply:
Applied suggested adjustment. Thank you.